net/can/can_callback.c

```diff
@@ -119,20 +119,6 @@ uint16_t can_callback(FAR struct net_driver_s *dev,
 
   if (conn)
     {
-      /* Try to lock the network when successful send data to the listener */
-
-      if (net_trylock() == OK)
-        {
-          flags = devif_conn_event(dev, flags, conn->sconn.list);
-          net_unlock();
-        }
-
-      /* Either we did not get the lock or there is no application listening
-       * If we did not get a lock we store the frame in the read-ahead buffer
-       */
-
-      if ((flags & CAN_NEWDATA) != 0)
-        {
 #ifdef CONFIG_NET_TIMESTAMP
           /* TIMESTAMP sockopt is activated,
```

Comment:
need remove the indent to confirm the code style

Reply:
this is a backport and I will not modify the original patch